cypress/e2e/pageObjects/OrangeHRMAddEmployee/actions.ts

```diff
@@ -62,7 +62,7 @@ class actions{
         cy.visit("https://opensource-demo.orangehrmlive.com/web/index.php/pim/addEmployee");
     }
 
-    NavigateToEmployeeListPage(){
+    NavigateToEmployeeListPage(){ 
         cy.visit("https://opensource-demo.orangehrmlive.com/web/index.php/pim/viewEmployeeList");
```

Comment:
For any URL you can add the **base URL (https://opensource-demo.orangehrmlive.com/web/index.php)**  TO CONFIG FILE and only add the rest of URL in everywhere you use URL